docs/metasploit-framework.wiki/Metasploit-Guide-HTTP.md

```diff
@@ -159,3 +159,30 @@ Module advanced options (auxiliary/scanner/http/title):
    VERBOSE               false                                              no        Enable detailed status messages
    WORKSPACE                                                                no        Specify the workspace for this module
 ```
+
+### HTTP Multiple-Headers
+Additional headers can be set via the ```HTTPRawHeaders``` option.
+A file containing a ERB template will be used to append to the headers section of the HTTP request.
+An example of an ERB template file is shown below.
+```
+Header-Name-Here: <%= 'content of header goes here' %>
+```
+
+The following output shows leveraging the scraper scanner module with an additional header stored in ```additional_headers.txt```.
+```
+msf6 auxiliary(scanner/http/scraper) > cat additional_headers.txt
```

Comment:
This mentions using an ERB template but then just defines the value as a static string. If you're going to mention that it's an ERB template it should probably be something that takes advantage of that functionality or it should just directly use the literal like:

```
Header-Name-Here: content of header goes here
```